requirements.txt

```diff
@@ -0,0 +1,15 @@
+paramiko>=2.7.2
+pandas>=1.3.2
+boxsdk>=2.11.0
+requests>=2.26.0
+pytest>=6.2.4
+git+https://github.com/AMP-SCZ/yaxil@main#egg=yaxil
```

Comment:
Why is it not `'yaxil>=0.5.2'` like that of setup.py?

Reply:
I haven't got to the `setup.py` the same way as `requirements.txt` (to use `AMP-SCZ` yaxil yet) Do you know how to structure `setup.py` to install from specific repository? We need to install `yaxil` under AMP-SCZ. (I had submitted a PR to `harvard-nrg/yaxil` but it has not been reviewed yet.)

Reply:
Yes I am working on the way.